src/Database/Dialect/MysqlDialectTrait.php

```diff
@@ -81,4 +81,12 @@ public function enableForeignKeySQL()
     {
         return 'SET foreign_key_checks = 1';
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function explainSQL($query)
```

Comment:
At first, I chose the name for the method. But I changed Sql to SQL, because in Driver class, we were using uppercase abbr. For example, [savePointSQL()](https://github.com/cakephp/cakephp/blob/db0638df2602d0d01086346685b104b678cf1e10/src/Database/Driver.php#L135)